### PR TITLE
Audit board sign in/out

### LINKED ISSUE #504: Allow audit board sign-in and sign-out
Blocked on #505.

### PR DESCRIPTION
**Pull request by ranweiler**
Closes #504.

Todo:
- [x] Indicate sign-in status in a way that is helpful to users who are not yet signed-in
- [x] Emit sign-in and sign-out notifications
- [x] Disable board sign-in button and page if audit not launched
- [x] Ensure that the audit button/navigation is disabled if the audit board is ever signed out

### CODE REVIEW

File: client/src/component/county/AuditBoard/SignedInPage.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+const SignedInPage = ({ auditBoard }: any) => {
+    return (
+        <div>
+            <CountyNav />
+            <div>
+                <h2>Audit Board Sign-in</h2>
+                <div className='pt-card'>
+                    Enter the full names and party affiliations of each member of
+                    the Acme County Audit Board who will be conducting this audit
```
Comment (kiniry) [inline client/src/component/county/AuditBoard/SignedInPage.tsx]: I presume this is a placeholder?

Comment (ranweiler) [reply client/src/component/county/AuditBoard/SignedInPage.tsx]: Yes, nice catch.

Comment (ranweiler) [reply client/src/component/county/AuditBoard/SignedInPage.tsx]: Fixed in 55ed72f.

Comment (kiniry) [reply client/src/component/county/AuditBoard/SignedInPage.tsx]: Thanks!

**Pull request merged**